flask_simplelogin/__init__.py

```diff
@@ -348,7 +349,16 @@ def login(self):
 
         return render_template("login.html", form=form, next=destiny), ret_code
 
+    def register_on_logout_callback(self, callback):
+        """Register a callback to be called on logout"""
+        self.on_logout_callbacks.append(callback)
+
     def logout(self):
         session.clear()
         self.flash("logout")
+
+        # call all registered callbacks
+        for callback in self.on_logout_callbacks:
```

Comment:
NIT this comment does not add much to the code, I would remove it

```suggestion
        for callback in self.on_logout_callbacks:
```